fetch-in-react-practice/client/src/components/Todos.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useEffect, useState } from 'react';
+import PageTitle from './PageTitle';
+import TodoList from './TodoList';
+import TodoForm from './TodoForm';
+
+export default function Todos() {
+  const [todos, setTodos] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  /* Implement useEffect to fetch all todos. Hints are at the bottom of the file. */
+  useEffect(() => {
+    async function getTodos() {
+      try {
+        const response = await fetch('/api/todos');
+        if (!response.ok) {
+          throw new Error(`Failed to fetch: ${response.status}`);
+        }
+        const currentTodoList = await response.json();
+        setTodos(currentTodoList);
+        setError(null);
```

Comment:
This shouldn't be necessary, but it doesn't hurt. However, the default value of `error` is `undefined`, so you should set it back to `undefined` instead of to `null`.